changelog/2336.removal.rst

```diff
@@ -0,0 +1 @@
+Removed support for Python 3.6
```

Comment:
Should we merge this with the 2.7 removal?

Reply:
Probably, although I wonder if it makes sense to keep the issues separate and then do this merge come release time so we capture each PR that contained the change?

Reply:
Sounds good, let's merge at release time